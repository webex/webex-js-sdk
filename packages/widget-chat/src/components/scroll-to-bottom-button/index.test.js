/* eslint max-nested-callbacks: ["error", 3] */
import React from 'react';
import renderer from 'react-test-renderer';
import ScrollToBottomButton from '.';

describe(`ScrollToBottomButton component`, () => {

  it(`basic renders properly`, () => {
    const component = renderer.create(
      <ScrollToBottomButton />
    );
    expect(component).toMatchSnapshot();
  });

  it(`with text renders properly`, () => {
    const component = renderer.create(
      <ScrollToBottomButton label="Test Label" />
    );
    expect(component).toMatchSnapshot();
  });

});
