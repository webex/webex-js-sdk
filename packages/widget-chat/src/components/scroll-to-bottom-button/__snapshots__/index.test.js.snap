exports[`ScrollToBottomButton component basic renders properly 1`] = `
<div
  className="scroll-to-bottom-container">
  <button
    className="scroll-to-bottom-button"
    onClick={undefined}>
    <span
      className="scroll-to-bottom-icon" />
  </button>
</div>
`;

exports[`ScrollToBottomButton component with text renders properly 1`] = `
<div
  className="scroll-to-bottom-container">
  <button
    className="scroll-to-bottom-button"
    onClick={undefined}>
    <span
      className="scroll-to-bottom-label">
      Test Label
    </span>
    <span
      className="scroll-to-bottom-icon" />
  </button>
</div>
`;
